### PR TITLE
Update readme.html

### PR DESCRIPTION
**Pull request by NatashaE**
Added topic: Laugh

### CODE REVIEW

**Review by AllstonS — COMMENTED** [review]
@NatashaE He sounds perfect for you, because you won't have to worry about eggs in your house:)

File: readme.html [hunk 1]
```diff
@@ -91,6 +91,9 @@ <h1 id="logo">
 
 <p>WordPress is the official continuation of <a href="http://cafelog.com/">b2/caf&#233;log</a>, which came from Michel V. The work has been continued by the <a href="https://wordpress.org/about/">WordPress developers</a>. If you would like to support WordPress, please consider <a href="https://wordpress.org/donate/" title="Donate to WordPress">donating</a>.</p>
 
+<h2>Laugh</h2>
```
Comment (amypiazza00) [inline readme.html:94]: Haha.

File: readme.html [hunk 2]
```diff
@@ -91,6 +91,9 @@ <h1 id="logo">
...
+<p>A programmer is going to the grocery store and his wife tells him, "Buy a gallon of milk, and if there are eggs, buy a dozen." So the programmer goes, buys everything, and drives back to his house. Upon arrival, his wife angrily asks him, "Why did you get 13 gallons of milk?" The programmer says, "There were eggs!"</p>
```
Comment (ghost) [inline readme.html:95]: I laughed (out loud)! Thanks for the contribution.

**Review by amypiazza00 — CHANGES_REQUESTED** [review]
Laughing.

Can you also add something about chuckling?

**Review by ghost — APPROVED** [review]
Laughed!

**Review by amypiazza00 — APPROVED** [review]

**Pull request merged**